### ISSUE TITLE
[DISCUSSION] We need a support forum for Goteo!

### ISSUE THREAD

**Issue by perpi** [Goteo/Goteo#138]
As Goteo is the only **social** crowdfounding service, it's needed to have a discussion forum for a better relation with users and also as Q&A service.
Currently, there's many free modern forum apps such as [Discourse](http://meta.discourse.org), [nodeBB](http://community.nodebb.org), [VanillaForums](http://vanillaforums.org/discussions) and [DuckDuckGo community platform](https://github.com/duckduckgo/community-platform) which can be used as a base for founding an english support forum.

In which context it can be helpful? Editing/Creating new themes, Installation, Development, Feedbacks, Having a desirous community, Supporting users, etc.

We need it, Please!

If money is problem, at the first stage it can be run on low-price/gratis host!

**Comment by cesc1989** [comment]
Agreed with @diff- It's a great approach to open source the platform but some kind of communication with original project devs may be necesary. Discourse may be the best choice though it'll need a server to be run.
